gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemConfiguration.java

```diff
@@ -106,6 +106,15 @@ public class GoogleHadoopFileSystemConfiguration {
   public static final HadoopConfigurationProperty<Boolean> GCS_HIERARCHICAL_NAMESPACE_ENABLE =
       new HadoopConfigurationProperty<>("fs.gs.hierarchical.namespace.folders.enable", false);
 
+  /**
+   * Configuration key for checksum writes
+   *
+   * <p>If this is enabled, write channel will calculate rolling checksum and compare it from server
+   * response.
+   */
+  public static final HadoopConfigurationProperty<Boolean> GCS_CHECKSUM_WRITE_ENABLE =
```

Comment:
For future extensibility, would it make sense for this to be multi-valued (e.g. enum)? For example, if GCS in the future adds a new kind of checksum, we'd be able to support it with a new value for the same configuration key instead of needing to add a second boolean property.

Reply:
Sounds good, will resolve this comment.